instagram/src/User.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+export default class User extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: {},
+      comment: {}
+    };
+  }
+  componentDidMount() {
```

Comment:
Same thing here. There's no actual need to store the user in `this.state`. Inside of the `div` tag, you can just access the user's name with `this.props.user.name`.